twisted/words/xish/xmlstream.py

```diff
@@ -14,6 +14,20 @@
 usable. See Twisted Words for Jabber specific protocol support.
 
 Maintainer: Ralph Meijer
+
+@var STREAM_CONNECTED_EVENT: Token indicating the connection has been
+    established.
+@type STREAM_CONNECTED_EVENT: L{intern}.
```

Comment:
There is no type `intern`, really. Just make this into `L{str}` (the native string).